src/components/VoteRow.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+const Wrapper = styled.div`
+  width: 150px;
+  margin: 10px;
+`;
+const Button = styled.button`
+  width: 100px;
+`;
+
+export default function VoteRow({ candidate, order, setCandidates }) {
+  const { id } = candidate;
+  const vote = async () => {
+    try {
+      const result = await axios.get(
```

Comment:
GET통신을 통해 받아오는 데이터는 무의미하므로 변수로 지정 안해주셔도 좋을 것 같습니다~!!
```suggestion
               await axios.get(
```